app/assets/javascripts/alchemy/alchemy.dragndrop.js.coffee

```diff
@@ -79,14 +79,14 @@ $.extend Alchemy,
 
   SortableContents: (selector, token) ->
     $(selector).sortable
-      items: "div.dragable_picture"
+      items: "div.draggable_picture"
```

Comment:
Should this class maybe also be called `sortable_picture`, in line with the other changes from `dragable` to `sortable`?